nav2_bringup/bringup/README.md

```diff
@@ -62,26 +62,31 @@ To view the robot model in RViz:
 
 ## Launch Navigation2 on a Robot
 
-Pre-requisites:
-* Run SLAM or Cartographer with tele-op to drive the robot and generate a map of an area for testing first. The directions below assume this has already been done. If not, it can be done in ROS1 before beginning to install our code.
+### Pre-requisites:
+* Run SLAM with Navigation 2 or tele-op to drive the robot and generate a map of an area for testing first. The directions below assume this has already been done or there is already a map of the area.
```

Comment:
You do _not_ require a map to use navigation

Reply:
@SteveMacenski Can you explain how nav2_bringup would work without a map? I am a bit confused here.

Reply:
Nothing in navigation should require, strictly speaking, any sort of map. 

We can navigate without a map - the costmaps dont require it. 
We can position without a map - tf doesn't require it. 
We can ... without a map image. A common usecase of navigation is to navigate in an open field / construction site where a map image doesnt exist but you have the costmaps to navigate with, that dont need to know anything about the environment prior, and the planner works in the costmap as it is aware

Reply:
I understand that Navigation2 should not require a map to work. However, currently, if the user launches Nav2_bringup without a map, it will not work with the default configurations. I don't know if it is different in Nav2 master, but at least in Nav2 Dashing, the map_server will throw a warning message and go into the error state if there is no map.

```
[map_server-2] [INFO] [map_server]: Configuring
[map_server-2] [WARN] []: Error occurred while doing error handling.
[map_server-2] [FATAL] [map_server]: Lifecycle node entered error state
```

Reply:
Ok that's fine, I'd just ask that you then include a little note mentioning that a map isn't strictly required. Keep in mind that some thing will then fail in the case of running nav with slam since there's no map server, and that's a totally valid way of bringing up nav

Reply:
Note is fine, except typo `costpmaps` -> `costmaps`